Demo/Demo/Forms/FormViewController.swift

```diff
@@ -0,0 +1,44 @@
+//
+//  FormViewController.swift
+//  Demo
+//
+//  Created by Toni Kocjan on 29/07/2021.
+//
+
+import UIKit
+import PovioKit
+
+class FormViewController<F: ValidationForm>: UIViewController {
+  let formView: ValidationFormView<F>
+  
+  init(form: F) {
+    formView = .init(validationForm: form)
+    super.init(nibName: nil, bundle: nil)
+    setupViews()
+  }
```

Comment:
Maybe better to not inject the form directly, but have it create inside content view? Thus we keep the current structure we use in projects, what do you think 🤔 ?

Reply:
I agree that in the actual project you would not do it like this. But as this is just a demo, I don't think it matters much to be honest?